README.md

```diff
@@ -188,6 +189,7 @@ By Tommy Buonomo ([Frenchie Games](https://frenchiegames.app/#/))
 
 - Save The Potato [source-code](https://github.com/imaNNeo/save_the_potato) - 🥇 Winner of [Flame Game Jam 3.0](https://itch.io/jam/flame-jam-3/results) - Rotate the shields and save the potato from incoming orbs! By [imaNNeo](https://imanneo.dev). [for Android or iOS](http://savethepotato.com)
 
+
```

Comment:
```suggestion
```

Reply:
@spydon Old flappy flame (flappy bird) game removed

Reply:
I think the suggestion is for removing the extra empty line

Reply:
@ufrshubham Made changes as suggested